cmd/output_csv.go

```diff
@@ -46,7 +46,10 @@ func NewOutputCsv() (oc *OutputCsv) {
 
 func NewOutputCsvFromFile(file *os.File) (oc *OutputCsv) {
 	oc = new(OutputCsv)
-	oc.writer = csv.NewWriter(file)
```

Comment:
I don't think this line should be deleted.

Reply:
Oh dear. You're definitely right on that... too fast and not enough testing. Will fix and then merge later. Thanks!